v1/python-sdk/tutorials/automl-with-azureml/forecasting-many-models/auto-ml-forecasting-many-models.ipynb

```diff
@@ -469,7 +468,9 @@
     "\n",
     "Reuse of previous results (``allow_reuse``) is key when using pipelines in a collaborative environment since eliminating unnecessary reruns offers agility. Reuse is the default behavior when the ``script_name``, ``inputs``, and the parameters of a step remain the same. When reuse is allowed, results from the previous run are immediately sent to the next step. If ``allow_reuse`` is set to False, a new run will always be generated for this step during pipeline execution.\n",
     "\n",
-    "> Note that we only support partitioned FileDataset and TabularDataset without partition when using such output as input."
+    "> Note that we only support partitioned FileDataset and TabularDataset without partition when using such output as input.\n",
+    "\n",
+    "> Note that we **drop column** \"Revenue\" from the dataset in this step as this is not relevant for forecasting with the dataset used in this example. **Please modify the logic based on your data**."
```

Comment:
Maybe, here we can make it clear that the logic is inside ./scripts/data_processing_tabular.py?

Reply:
We have mentioned it a little above, here is the entire description:
![image](https://user-images.githubusercontent.com/74648335/206751693-06a206ab-0a32-497f-b4b4-d04417ddd711.png)